receiver/skywalkingreceiver/skywalkingproto_to_traces.go

```diff
@@ -31,15 +31,16 @@ import (
 )
 
 const (
-	AttributeRefType                  = "refType"
-	AttributeParentService            = "parent.service"
-	AttributeParentInstance           = "parent.service.instance"
-	AttributeParentEndpoint           = "parent.endpoint"
-	AttributeSkywalkingSpanID         = "sw8.span_id"
-	AttributeSkywalkingTraceID        = "sw8.trace_id"
-	AttributeSkywalkingSegmentID      = "sw8.segment_id"
-	AttributeSkywalkingParentSpanID   = "sw8.parent_span_id"
-	AttributeNetworkAddressUsedAtPeer = "network.AddressUsedAtPeer"
+	AttributeRefType                   = "refType"
```

Comment:
Do these const need to be public?

Reply:
It's for extensions, I can simply import pkg and try to match ID value in `trace data` by the attribute key defined here.

Reply:
> It's for extensions, I can simply import pkg and try to match ID value in `trace data` by the attribute key defined here.

@taloric this is also the reason why it started as `public`. 
BTW, If it needs to be considered as a dependency package, you can move `skywalkingproto_to_traces.go` into `[pkg](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/pkg)/[translator](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/pkg/translator)/` like jaeger translator

Reply:
> @taloric this is also the reason why it started as `public`. BTW, If it needs to be considered as a dependency package, you can move `skywalkingproto_to_traces.go` into `pkg/translator` like jaeger translator


That's a good idea, but I'm not sure yet .... because I notice that `skywalkingreceiver` stability status marked as `beta` currently (and I agree with this), so I think maybe we need more using cases and try to figure out best practice about it

Reply:
make sense, leave this in `skywalkingreceiver ` is fine.  and make those const to be public, temporarily.